rest/taskana-rest-spring/src/test/java/pro/taskana/user/rest/UserControllerIntTest.java

```diff
@@ -180,7 +183,7 @@ void should_ReturnExistingUsers_When_ParameterContainsDuplicateAndInvalidIds() t
   }
 
   @Test
-  void should_CreateValidUser_When_CallingCreateEndpoint() throws Exception {
```

Comment:
Why did you erase the "throws Exception"

Reply:
Because they were never thrown